_dprhtml/js/search.js

```diff
@@ -778,29 +778,29 @@ function createTables(xmlDoc,hiert)
 
                 tempexword = [];
 
-                if(DPR_G.G_searchRX) startmatch = findRegEx(texttomatch,getstring);

```

Comment:
are lines 626/627 required anymore?

Reply:
Good call. I'll remove them.

Reply:
if those lines were commented out - it would fix the issue but would regress another scenario which is completely escaping me. 

can you please make a timeboxed effort to guess what that is and ensure it is not being regressed.

from my understanding of your changes - you aren't writing to texttomatch in any new way - so that regression should not happen.

Reply:
@parthopdas So my original attempt at fixing the bug was to uncomment these lines. The regression was that the search results would no longer show bolded items. Here's an example screenshot depicting the regression:

# Before Uncommenting Lines
<img width="1293" alt="Screen Shot 2021-03-25 at 6 13 07 PM" src="https://user-images.githubusercontent.com/5777094/112550506-db128680-8d95-11eb-8321-bb0560ce3c2d.png">

# After Uncommenting Lines
<img width="1293" alt="Screen Shot 2021-03-25 at 6 13 12 PM" src="https://user-images.githubusercontent.com/5777094/112550500-d8b02c80-8d95-11eb-92bf-b02f9f0c8842.png">

Reply:
Wonderful wonderful. That is all what I wanted to know.

Reply:
Please push through!